generators/entity-server/templates/src/main/java/package/repository/EntityRepository.java.ejs

```diff
@@ -37,6 +37,10 @@ import org.springframework.data.repository.query.Param;
 import org.springframework.data.mongodb.repository.Query;
 import org.springframework.data.mongodb.repository.MongoRepository;
 <%_ } _%>
+<%_ if (databaseType === 'neo4j') { _%>
+import org.neo4j.springframework.data.repository.Neo4jRepository;
+import java.util.List;
```

Comment:
I think this import is now obsolete (because of the workaround removal of https://github.com/neo4j/sdn-rx/issues/51)

Reply:
Still needed

```
List<User> findAllByActivatedIsFalseAndActivationKeyIsNotNullAndCreatedDateBefore
```

Reply:
Isn't that only in the `UserRepository`? I think the repository for an entity the list import should come from `  let importList = fieldsContainOwnerManyToMany;` property. But that's details. Eventually organizing imports will fix it, we have some not used imports somewhere else too.